data/managed/log_sources/cloudflare/log_source.yml

```diff
@@ -0,0 +1,21 @@
+name: cloudflare
+
+ingest:
+  select_table_from_payload_metadata: |
+    if match(.__metadata.s3.key, r'audit') {
+      "audit"
+    } else if match(.__metadata.s3.key, r'dns') {
+      "dns"
+    } else if match(.__metadata.s3.key, r'firewall_event') {
+      "firewall_event"
+    } else if match(.__metadata.s3.key, r'http_request') {
+      "http_request"
+    } else if match(.__metadata.s3.key, r'nel_report') {
+      "nel_report"
+    } else if match(.__metadata.s3.key, r'network_analytics') {
+      "network_analytics"
+    } else if match(.__metadata.s3.key, r'spectrum_event') {
+      "spectrum_event"
+    } else {
+      "default" # will error table not found
```

Comment:
why err and not abort, e.g. will it cause error when Cloudflare sends verification file ? : https://developers.cloudflare.com/logs/get-started/enable-destinations/aws-s3/#manage-via-the-cloudflare-dashboard

Reply:
lets abort